tests/lib/tpp_schema.py

```diff
@@ -309,6 +309,13 @@ class APCS_JRC20231009_LastFilesToContainAllHistoricalCostData(Base):
     )
 
 
+class AllowedPatientsWithTypeOneDissent(Base):
```

Comment:
am I misunderstanding, or should this be *without*? 

```suggestion
class AllowedPatientsWithoutTypeOneDissent(Base):
```

Reply:
(although if that's the tablename TPP have used, then, obvs, it's that...!)

Reply:
Yeah, I think they've just prefixed "Allowed" to the previous table name to indicate the change from deny-list to allow-list – though I agree the name is now somewhat misleading!

Reply:
maybe it's worth a comment above the class def?

Reply:
Yeah fair point, although it can't really go in the class def because that file is autogenerated from the db schema. But I can add a comment at the point in the backend where it's used.